apps/client/server/profile/mutations.ts

```diff
@@ -0,0 +1,13 @@
+import type { UseQueryOptions } from "@tanstack/react-query";
+import { useMutation } from "@tanstack/react-query";
```

Comment:
```suggestion
import { useMutation, type UseQueryOptions } from "@tanstack/react-query";
```

한줄로도 가능합니다! 개바개이므로 입맛에 맞으시면 사용해주세요!